src/components/Testimonials/index.js

```diff
@@ -11,126 +11,122 @@ import {
 import Heading from '../Heading';
 import './style.css';
 
-class Testimonials extends Component {
-  state = {
-    data: {},
-    error: false,
-    loading: true
-  };
+const Testimonials = () => {
+  const [data, setData] = useState({});
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
 
-  componentDidMount() {
+  useEffect(() => {
     charityAPI({ url: '/What-they-say' })
```

Comment:
can you put line 20:29 in a getData function and then use it int the useEffect Hooks

Reply:
ok I will